tests/test_extract.py

```diff
@@ -1,38 +1,41 @@
 from datetime import datetime, timedelta
 import os
-from unittest import skip
 
 import polib
-from pytz import UTC
 
-from i18n import extract
+from i18n import extract, config
 
-from . import I18nToolTestCase
+from . import I18nToolTestCase, MOCK_DJANGO_APP_DIR
 
 # Make sure setup runs only once
 SETUP_HAS_RUN = False
 
 
-# Skip this test because it takes too long (>1 minute)
-# TODO: figure out how to declare a "long-running" test suite
-# and add this test to it.
-@skip('Long running test')
 class TestExtract(I18nToolTestCase):
     """
     Tests functionality of i18n/extract.py
     """
     generated_files = ('django-partial.po', 'djangojs-partial.po', 'mako.po')
 
-    def setUp(self):
+    def setUp(self, root_dir=MOCK_DJANGO_APP_DIR, preserve_locale_paths=None, clean_paths=None):
```

Comment:
This convention of adding arguments to setUp seems strange to me.  Isn't there a better way to do this?  I haven't seen this done elsewhere.

Reply:
Sure, I can add those elsewhere. I'll clean up that up along with the rest shortly.